process/datadog_checks/process/process.py

```diff
@@ -242,7 +242,7 @@ def psutil_wrapper(self, process, method, accessors, try_sudo, *args, **kwargs):
                 process_ls = subprocess.check_output(ls_args)
                 result = len(process_ls.splitlines())
             except Exception as e:
-                self.log.exception("Trying to retrieve %s with sudo failed with error: %s", method, e)
```

Comment:
Idem.

Reply:
Oops this was a mistake from an older change I made.